### PR TITLE
feat: implement title component

### PR DESCRIPTION
**Pull request by kenanchristian**

### CODE REVIEW

File: workspaces/lluis/Title.svelte [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  export let size = isSubtitle ? 5 : 3
+</script>
+
+<h1
+  class:title={!isSubtitle}
+  class:subtitle={isSubtitle}
+  class="{`is-size-${size} is-size-${sizeMobile || size}-mobile is-size-${sizeTablet || size}-tablet is-size-${sizeDesktop || size}-desktop`}"
+  class:is-spaced={isSpaced}>
+  <slot />
+</h1>
```
Comment (kantord) [inline workspaces/lluis/Title.svelte:16]: Can you please import the stylesheet here, and remove it at the original place?

**Review by kantord — CHANGES_REQUESTED** [review]
Hi @kenanchristian , thanks for the PR!

Looks good to me more or less. Are there any remaining instances of the title class that are not using this new component? If not, then I think it should be ready

File: workspaces/lluis/Title.svelte (hunk 1 above)
Comment (kenanchristian) [reply workspaces/lluis/Title.svelte:23]: Done! I also added support for text alignment and weight

**Comment by kantord** [comment]
There is now a conflict in 
workspaces/web/src/mystyles.scss, can you please resolve it? :pray:

**Comment by kantord** [comment]
> There is now a conflict in
> workspaces/web/src/mystyles.scss, can you please resolve it? pray

never mind, it was trivial to resolve with the GUI so I did :-)

**Comment by kenanchristian** [comment]
> > There is now a conflict in
> > workspaces/web/src/mystyles.scss, can you please resolve it? pray
> 
> never mind, it was trivial to resolve with the GUI so I did :-)

Ah sorry, I missed that when pushing. Thank you for resolving it :)

**Comment by kantord** [comment]
Thanks @kenanchristian!

@allcontributors please add @kenanchristian for code

**Pull request merged**